common/components/utils/url.js

```diff
@@ -169,7 +169,7 @@ class urlHelper {
       return _url;
     } else {
       let _args: Dictionary<string> = {
-        prev: sectionArgs.section,
+        prev: sectionArgs.section!==Section.IframeProject ? sectionArgs.section : Section.AboutProject, // if login from within an iframe, go direct without iframe
```

Comment:
Do we want to have the same logic for groups?

Also, be sure to add new `logInThenReturn` unit tests to https://github.com/DemocracyLab/CivicTechExchange/blob/3f6985642c0992789d331eefc899ae8231b562ea/common/components/test/utils.test.js#L157